DashJSInTypeScriptTut/Using-DashJS-with-TypeScript-in-VSCode.md

```diff
@@ -1,116 +1,117 @@
+---
 summary: Using DashJS with TypeScript in VSCode
 id: Using-DashJS-with-TypeScript-in-VSCode
 categories: Typescript
 tags: medium
 status: Published
 authors: Rion, Schlomon
 Feedback Link: example.com
+---
```

Comment:
Putting this line here somehow renders a new page in codelab saying "Feedback Link: example.com"

Reply:
Just added another commit to the PR that hopefully fixes this.  I think it may be because "Feedback Link" (with capital letters) wasn't a recognized reserved metatag when using markdown (it would be for Google Doc).

Reply:
https://github.com/googlecodelabs/tools/blob/master/FORMAT-GUIDE.md#formatting-reference